PRODUCTIONISATION.md

```diff
@@ -27,7 +27,7 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 ### 1. General requirements
 
 * [ ] 1.1 Application source code hosted with a Git VCS service
-* 1.2 Adhere to project coding standards and code style guideline
+* 1.2 Adhere to project coding standards
   * [ ] 1.2.1 Provides a style guideline
   * [ ] 1.2.2 Consistency within development team
```

Comment:
Unsure of what this point is? Is this to be checked via a linting tool to ensure it conforms to a code style?

Reply:
Yes. An IDE, Rider for example, is also capable of doing this for certain languages such as C# where it is picked up automatically.

Reply:
I think this is implied by  having "coding standards", and the newly added "enforce coding standards", so can safely be removed

Reply:
Resolved in https://github.com/madetech/productionisation/pull/51/commits/673ffdc7799b26c4983a3f999b73d368740afbf8.